src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -568,6 +637,21 @@ private static void handleKeyValueConfig(TestInputConfiguration.Builder testInpu
         }
     }
 
+    public static Map<String, String> getDefaultProperties(String fullyQualifiedClassName) {
+
+        final Map<String, String> defaultProperties = new HashMap<>();
+        final boolean isModuleValid = !SUPPRESSED_MODULES.contains(fullyQualifiedClassName);
+        final ModuleDetails moduleDetails = ModuleDetailsMapHolder.PUBLIC_MODULE_DETAILS_MAP
+                .get(fullyQualifiedClassName);
+
+        if (isModuleValid && moduleDetails != null) {
+            moduleDetails.getProperties().forEach(prop -> {
+                defaultProperties.put(prop.getName(), prop.getDefaultValue());
+            });
+        }
+        return defaultProperties;
+    }
+
     private static String getFullyQualifiedClassName(String filePath, String moduleName)
             throws CheckstyleException {
         String fullyQualifiedClassName;
```

Comment:
here is no new line before, so its questionable doing some and others done. Not seen this in https://www.oracle.com/docs/tech/java/codeconventions.pdf ether.

Reply:
Let's not touch what is not required.
We need to merge PR sooner.

Reply:
yes of course, just want to point our the issue.

Reply:
dont know about that, yes then its nice but big migration.